### PR TITLE
add new command util

### PR DESCRIPTION
**Pull request by alibazregar**

### CODE REVIEW

File: hostManager.js [hunk 1]
```diff
@@ -91,6 +92,9 @@ export default class HostManager {
    * @returns {HostEndPoint}
    */
   #createHttpEndPoint(name, options, service) {
+    let externalCommands = LoadCommand.processSync(
+      service.settings._options.Settings.LibPath
+    );
```
Comment (Qamsari) [inline hostManager.js:97]: Command or External command must load in service not in create endpoint! exactly only in `HostService` class.

**Review by Qamsari — COMMENTED** [review]
Please see my issue

File: renderEngine/Command/Collection/CollectionCommand.js [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
-  constructor(collectionCommandIl) {
-    super(collectionCommandIl);
-    this.commands = collectionCommandIl["Commands"].map(
-      CommandUtil.createCommand
-    );
+  constructor(collectionCommandIl, externalCommands) {
+    super(collectionCommandIl,externalCommands);
+    this.commands = collectionCommandIl["Commands"].map((command) => {
+      CommandUtil.createCommand(command, externalCommands);
+    });
```
Comment (Qamsari) [inline renderEngine/Command/Collection/CollectionCommand.js:17]: Do you have any idea about remove `externalCommands` from `ctor` parameter of `CollectionClass` and her sub class?
I Send voice for you about it.

**Review by Qamsari — COMMENTED** [review]
please see my issues. (Children of Israel issues 🤣 )

File: Models/Index1Response.js [hunk 3]
```diff
@@ -15,13 +15,14 @@ export default class Index1Response extends RequestBaseResponse {
...
+   * @param {Object.<string, any>} externalCommands
    *  @returns {Promise<[number,NodeJS.Dict<number | string | string[]>,*]>}
    */
   async getResultAsync() {
     try {
       const commandIl = JSON.parse(this._request.cms.page_il);
-      const command = CommandUtil.createCommand(commandIl);
-      const context = new RequestContext(this._settings, this._request);
+      const command = CommandUtil.createCommand(commandIl, externalCommands);
+      const context = new RequestContext(this._settings, this._request,externalCommands);
```
[the remark this answers is not in the document]
Comment (alibazregar) [reply Models/Index1Response.js]: fixed

**Review by Qamsari — COMMENTED** [review]
please resolve conflict

**Pull request merged**